### ISSUE TITLE
Debugger breakpoint not displayed well in block

### ISSUE THREAD

**Issue by ericwinger** [GemTalk/Jadeite#134]
- Compile the method ``interestingLoop`` below in a test class. 
- Set a breakpoint on the sender of ``test``
- execute the method to bring up a debugger

Note the difference in the breakpoint display depending on which frame is selected. the block context is wrong. 

![image](https://user-images.githubusercontent.com/36212516/44886424-ec955f80-ac7a-11e8-90ba-41ff5c579ce5.png)

![image](https://user-images.githubusercontent.com/36212516/44886426-f028e680-ac7a-11e8-8c22-083195ee5516.png)

```
interestingLoop

	| contents result |
	result := String new. 
	contents := { 23 . 55 . 45 . 999 }.
	contents select: [:n |
		| temp |
		temp := self test + n. 
		(temp / self anotherNumber) > 5
			ifTrue:[result add: 'found']
			ifFalse: [result add: 'not found'].
		result add: ' - '.
		n = 2.
	].
	^result

**Comment by ericwinger** [comment]
When you're in an executed block frame, it appears the breakpoint source range is being sent back wrong from the server. Want to see what 3.4 server sends back when that version is supported in Rowan.

**Comment by ericwinger** [comment]
Should have been fixed in release `3.0.85`

**Comment by LisaAlmarode** [comment]
This was fixed as part of some other complaints a while back; things look fine in 3.0.92
